halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/oraclebmcs/OracleBMCSAccount.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2017 Oracle America, Inc.
+ *
+ * The contents of this file are subject to the Apache License Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * If a copy of the Apache License Version 2.0 was not distributed with this file,
+ * You can obtain one at https://www.apache.org/licenses/LICENSE-2.0.html
+ */
+
+package com.netflix.spinnaker.halyard.config.model.v1.providers.oraclebmcs;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
+import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class OracleBMCSAccount extends Account {
+  private String compartmentId;
+  private String userId;
+  private String fingerprint;
+  private String sshPrivateKeyFilePath;
```

Comment:
I would annotate this with `@LocalFile` - this ensures it will be mounted as a required config file when doing a remote deployment.